### ISSUE TITLE
⌨ issue with skills picker

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15650]
### ⌨ Accessibility and Expected Behaviour

**🛡️Fable user reported:** 
The dropdown edit field to search for a skill and select an option is inaccessible. After I enter a skill, I'm unable to access the list of options that have populated. When I swipe past the field, I see that there is a match, but there is no way to see any of the results.

### WCAG 2.1 Criteria

2.1.1 Keyboard

### 📋 Steps to Reproduce

1. sign in
2. navigate to add an experience 
3. add a new experience 
4. add a skill to the experience

### What operating systems are you encountering the problem on?

IOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver
